src/components/ArticleId.jsx

```diff
@@ -26,14 +27,26 @@ function ArticleId() {
         setIsCommentsLoading(true);
         fetchComments(article_id)
           .then((comments) => {
-            console.log(comments);
-            setComments([...comments]);
+              const formattedComments = comments.map(comment => {
+                  const formattedDate = new Date(comment.created_at)
+                  return {...comment, formatted_date: formattedDate.toLocaleString("en-GB")}
+              })
+
+            setComments([...formattedComments]);
             setIsCommentsLoading(false);
           });
       }, [article_id]);
 
     const Comment = <section className="comments"> 
-        { isCommentsLoading ? <Loading /> : <p>{comments.length}</p> }
+        {isCommentsLoading ? <Loading /> : comments.map((comment) => {
+            return <div className="comment_row" key={comment.comment_id}>
+                <p className="comment_body">{comment.body}</p>
+                <div className="comment_meta">
+                    <span className="comment_meta_item">{"Votes: "+comment.votes}</span>
+                    <span className="comment_meta_item">{comment.formatted_date}</span>
+                    <span className="comment_meta_item">{comment.author}</span></div>
```

Comment:
For these sections of each comment, it might be worth adding a bit more detail so users can understand what each thing represents, for example: instead of just stating the author's name on its own, put something like 'Posted by <author>' and instead of a random date, put something like 'Posted on <date>'